### DISCUSSION TITLE
create container component with svg

### DISCUSSION THREAD

**Discussion by d-opri** [neuefische/questions#535] [Web] [answered]
i have a svg that i would like to use as the backround for a container/card component. i want to display the text on top and have it be responsive. i tried inserting the svg in a container, setting the z-index to -1, positioning absolute, but nothings working for me. i managed to position the svg behind the text but theyre "not connected" so the texts moves outside of the svg when longer.

thanks!

**Comment by mbosselmann** [comment, marked as answer]
Hi Danie,
can you post a link to the current branch? Then we can give you a better response for your question. 👀

**Reply by mbosselmann** [reply]
We looked into Danie's code together and decided to not use a SVG for this case.
